probes/packagedNpmWithGitHubWorkflow/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: packagedNpmWithGitHubWorkflow
```

Comment:
We check for more than just `npm` packaging methods? 

You can see a list of them at:
https://github.com/ossf/scorecard/blob/5b8a33ddea7c923ec05843bd18fa1dbe5612c6fc/checks/fileparser/github_workflow.go

Is this probe supposed to be npm forcused, or do we expect to have one probe for each method? Are we going to generalize any? @laurentsimon

Reply:
Maybe a better name could be along the lines of `packagedWithAutomatedScript`?

Reply:
start with general probe, can always be more granular later.

Reply:
Renamed in https://github.com/ossf/scorecard/pull/3486/commits/148801e5b24c381a0b3549caa46f75597a42b690